h2d/Layers.hx

```diff
@@ -12,14 +17,25 @@ class Layers extends Object {
 		layerCount = 0;
 	}
 
+	/**
+		Adds a child object at the bottom layer (layer 0).
+	**/
 	override function addChild(s) {
 		addChildAt(s, 0);
 	}
 
+	/**
+	 * Shorthand for `addChildAt`, see `h2d.Layers.addChildAt`.
+	 */

```

Comment:
I think that it should be the other way : add is the correct API function and addChildAt is a substitute. Because addChildAt already has its own definition in h2d.Object, which slightly differs in semantics in h2d.Layers.

Reply:
Fixed